workflows/files/ZWECONF.xml

```diff
@@ -12,9 +12,9 @@
     </workflowInfo>
     <!--variables-->
     <variable name="zowe_setup_dataset_prefix" scope="instance" visibility="public">
-        <label>Zowe setup MVS prefix</label>
-        <abstract>Where Zowe MVS data sets will be installed</abstract>
-        <description>Where Zowe MVS data sets will be installed</description>
+        <label>Prefix of existing Zowe Target libraries</label>
+        <abstract>Prefix of existing Target libraries created during PSWI or SMP/E</abstract>
+        <description>Prefix of existing Target libraries created during PSWI or SMP/E installation phase</description>
```

Comment:
Add here that it is HLQ of SZWEEXEC, please.

Reply:
updated